### PR TITLE
Flake8 all the things

### PR DESCRIPTION
**Pull request by blkperl**
Most of the changes were done by `autopep8 -i -r . --ignore E501`

It's easier to review if you look at it without the whitespace changes: https://github.com/18F/domain-scan/pull/57/files?w=1

### CODE REVIEW

**Comment by konklone** [comment]
Thank you! I like almost all of these changes -- the one thing I don't like (or agree with in pep8) is the line length limitation. If you can rerun flake8 and ignore the constaint that forces line wrapping, I'd be very happy to merge.

File: scanners/subdomains.py [hunk 1]
```diff
@@ -141,25 +140,22 @@ def scan(domain, options):
...
     # bad hostname for cert?
-    if (protocol == "https") and (endpoint.get("https_bad_name", False) == True):
-        bad_cert_name = True
+    if (protocol == "https") and (endpoint.get("https_bad_name", False) is True):
+        bad_cert_name = True  # nopep8
     else:
-        bad_cert_name = False
-
-
+        bad_cert_name = False  # nopep8
```
Comment (blkperl) [inline scanners/subdomains.py]: pep8 is complaining that this variable is never used. I'm not sure why this code is here so I silenced pep8

**Comment by blkperl** [comment]
@konklone Okay done!

**Comment by konklone** [comment]
Thanks, @blkperl! I've gone and turned on Travis, too.

**Pull request merged**